homeassistant/components/rollershutter/zwave.py

```diff
@@ -54,6 +54,12 @@ def value_changed(self, value):
             _LOGGER.debug("Value changed on network %s", value)
 
     @property
+    def name(self):
```

Comment:
Does this work in real life? because it is the superior zwave class that names the device. Also the linter complains that `node` is undefined. Please check. I do not think we will get this function merged into HA, because the user should use the `customize:` section of the config to rename friendly names, and use ozwcp to rename node names.